roledb.go

```diff
@@ -293,19 +294,38 @@ func ReadRawGroups(importedGroups []RawAuthGroup) ([]AuthGroup, error) {
 // The 'cache' parameter is used to speed up subsequent calls to this function,
 // because this function tends to get used in loops.
 func GroupIDsToNames(groups []GroupIDU32, db RoleGroupDB, cache map[GroupIDU32]string) ([]string, error) {
-	names := make([]string, len(groups))
-	for i, gid := range groups {
+	// A group in 'groups' may not exist in the RoleGroupDB and we don't want to return a blank item in the string array.
```

Comment:
This comment only makes sens if someone sees the previous code. I don't think we need it.